### PR TITLE
Update Developer's Guide for Add Feature

### LINKED ISSUE #95: Add Add feature to Developer Guide
Description of implementation, Rationale of implementation, with code snippets and UML diagrams to aid your explanations.

### PR DESCRIPTION
**Pull request by Kok-je**
Fixes #95 

Updated UML diagram attached in file.

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -237,6 +237,27 @@ The following activity diagram summarizes what happens when a user executes a ne
...
+
+This section explains the implementation of the `add` feature. The command takes in two compulsory parameters which is the employee name and position, executing the command leads to the addition of an employee person into the records of coydir.
+
+Below is a sequence diagram and explanation of how `add` is executed.
+
+<img src="images/AddCommandUML.png" width="550" />
+
+Step 1. The user enters the command `add n/Jon j/janitor`
+
+Step 2. User input is parsed by `AddCommandParser` which creates the`AddCommand` object, then the method LogicManager#execute is called to create the AddCommand object.
```
Comment (prit3010) [inline docs/DeveloperGuide.md:252]: Perhaps can go a bit more into detail. Like AddCommandParser creates the AddCommand object while the execute function in AddCommand creates a CommandResult Object

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -237,6 +237,27 @@ The following activity diagram summarizes what happens when a user executes a ne
...
+
+#### Implementation
+
+This section explains the implementation of the `add` feature. The command takes in two compulsory parameters which is the employee name and position, executing the command leads to the addition of an employee person into the records of coydir.
+
+Below is a sequence diagram and explanation of how `add` is executed.
+
+<img src="images/AddCommandUML.png" width="550" />
+
+Step 1. The user enters the command `add n/Jon j/janitor`
```
Comment (prit3010) [inline docs/DeveloperGuide.md:250]: Missing fullstop

**Comment by chao890** [comment]
Maybe can standardize the font sizes used in the UML diagram, but otherwise looks good.

**Review by prit3010 — COMMENTED** [review]
Could be a bit more detailed on the implementation side and missing full stops but not very important as of now. LTGM

**Review by kevinchangjk — APPROVED** [review]
LGTM.

**Pull request merged**